triggers.yml.example

```diff
@@ -1,40 +1,120 @@
-#Example file for repository virtio-win/kvm-guest-drivers-windows
+# Example file for repository virtio-win/kvm-guest-drivers-windows
 # https://github.com/virtio-win/kvm-guest-drivers-windows
+#
+# <driver>:
+#   include:
+#     - <path_to_file>
+#     - <path_to_dir>/
+#   exclude:
+#     - <path_to_file>
+#     - <path_to_dir>/
+#
+# Notes:
+#   Directory should have / at the end
+
 Balloon:
-  - 'Balloon'
-Documentation:
+  include:
+    - Balloon/
+  exclude:
+    - Balloon/app/
+    - Balloon/LICENSE
 NetKVM:
-  - 'NetKVM'
-Q35:
-Tools:
-  - '*'
-VirtIO:
-  - '*'
-fwcfg:
+  include:
+    - NetKVM/
+  exclude:
+    - NetKVM/DebugTools/
+    - NetKVM/Documentation/
+    - NetKVM/LICENSE
 fwcfg64:
-  - 'fwcfg64'
+  include:
+    - fwcfg64/
 ivshmem:
-  - 'ivshmem'
-packaging:
-  - '*'
-pciserial:
+  include:
+    - ivshmem/
+  exclude:
+    - ivshmem/test/
 pvpanic:
-  - 'pvpanic'
+  include:
+    - pvpanic/
+  exclude:
+    - pvpanic/PVPanic Package/
+    - pvpanic/LICENSE
 viocrypt:
-  - 'viocrypt'
+  include:
+    - viocrypt/
+  exclude:
+    - viocrypt/dll/
+    - viocrypt/package/
+    - viocrypt/test/
+    - viocrypt/LICENSE
+viofs:
+  include:
+    - viofs/
+  exclude:
+    - viofs/VirtFS Package/
+    - viofs/svc/
+viogpu:
+  include:
+    - viogpu/
+  exclude:
+    - viogpu/viogpuap/
+    - viogpu/viogpusc/
 vioinput:
-  - 'vioinput'
+  include:
+    - vioinput/
+  exclude:
+    - vioinput/LICENSE
 viorng:
-  - 'viorng'
+  include:
+    - vioinput/
+  exclude:
+    - viorng/VirtRNG Package/
+    - viorng/cng/um/
+    - viorng/test/
+    - viorng/compile.readme
 vioscsi:
-  - 'vioscsi'
+  include:
+    - vioscsi/
+  exclude:
+    - vioscsi/LICENSE
 vioserial:
-  - 'vioserial'
+  include:
+    - vioserial/
+  exclude:
+    - vioserial/app/
+    - vioserial/benchmark/
+    - vioserial/lib/
+    - vioserial/libtestapp/
+    - vioserial/libtestsvc/
+    - vioserial/LICENSE
 viosock:
-  - 'viosock'
+  include:
+    - viosock/
+  exclude:
+    - viosock/ViosockPackage/
+    - viosock/lib/
+    - viosock/viosock-test/
+    - viosock/viosocklib-test/
+    - viosock/wsk/
+    - viosock/LICENSE
 viostor:
-  - 'viostor'
-viogpu:
-  - 'viogpu'
-/:
-  - '*'
+  include:
+    - viostor/
+  exclude:
+    - viostor/LICENSE
+"*":
+  include:
+    - /
+    - Tools/
+    - VirtIO/
+    - packaging/
```

Comment:
Same here. I think packaging projects should be included.

Reply:
We discussed it off the list. Packaging projects should be added.

Reply:
Fixed